### ISSUE TITLE
Coursera｜150门免费实践项目

### ISSUE THREAD

**Issue by ruimaruima** [ruimaruima/mooc#26]
# Coursera｜150门免费实践项目

Coursera提供了150门免费的小型实践项目, 截止日为为2020年6月12日. 这些项目通常收费为$10. 

实践项目

These “Guided Projects” as Coursera calls them, are offered via Rhyme, which was Coursera’s first acquisition. They give you access to a virtual desktop which comes with all the appropriate software needed to do the project, while watching a video side by side.

1. 在Visual Studio中使用C＃构建计算器
Building a Calculator using C# in Visual Studio

2. Google AI平台上的自定义预测实例
Custom Prediction Routine on Google AI Platform

3. 工作中的项目和时间安排
Intro to Scheduling with When I Work

4. 
Object-Oriented Programming with Java

5. 
Web Application Security Testing with OWASP ZAP

6. 
Gmail: The Foundation To Accessing Google Apps

7. 
Use Mailchimp to Build an E-mail Marketing Campaign

8. 
Build a Deep Learning Based Image Classifier with R

9. 
User Experience Design – Creating User Profiles

10. 
Basic Image Classification with TensorFlow

11. 
Create Your First Python Program

12. 
Build a Persistent Storage App in Android Studio

13. 
RESTful API with HTTP and JavaScript

14. 
Predict Diabetes with a Random Forest using R

15. 
Predict Housing Prices in R on Boston Housing Data

16. 
Building Digital Media using Graphic Design in Google Slides

17. 
Build a Linear Layout App in Android Studio
Beginning SQL Server
Using Effcient Sorting Algorithms in Java to Arrange Tax Data
Generate Synthetic Images with DCGANs in Keras
Dimensionality Reduction using an Autoencoder in Python
Introduction to Project Management
Building a Text-Based Bank in Java
Build Data Analysis and Transformation Skills in R using DPLYR
Improve Business Performance with Google Forms
Compare Stock Returns with Google Sheets
Build a Simple App in Android Studio with Java
Create Employee Management System with When I Work
Anomaly Detection in Time Series Data with Keras
Portfolio Optimization using Markowitz Model
Building Basic Relational Databases in SQL Server Management Studio
Predict Future Product Prices Using Facebook Prophet
Computer Vision – Object Tracking with OpenCV and Python
Client Booking & Scheduling with Picktime
Hierarchical Clustering: Customer Segmentation
Use WordPress to Create a Blog for your Business
Build a Data Science Web App with Streamlit and Python
Develop a Company Website with Wix
Create Relational Database Table Using SQLiteStudio
Create a Resume and Cover Letter with Google Docs
Image Compression and Generation using Variational Autoencoders in Python
Build Your Portfolio Website with HTML and CSS
Build a Relative Layout App in Android Studio
Applying Data Structures to Manipulate Cleansed UN Data
Practical Introduction to the Command Line
Predict Gas Guzzlers using a Neural Net Model on the MPG Data Set
Create Interactive Dashboards with Streamlit and Python
Analyzing Company’s Performance using Ratios
Neural Network Visualizer Web App with Python
User Experience Responsive Hero in XD for Beginners
Retrieve Data using Single-Table SQL Queries
Build a Full Website using WordPress
Build a Machine Learning Web App with Streamlit and Python
Retrieve Data from Multiple-Table SQL Queries
Generando modelos con Auto Machine Learning
Web Application Security Testing with Burp Suite
Hiring and Onboarding Employees with ClickUp
Use Canva to Create Social Media Marketing Designs
Using Shiny to Plot Differential Gene Expression
Create and Format a Basic Document with LibreOffice Writer
Introduction to Google Docs
Introduction to Project Management with ClickUp
Compose and Program Music in Python using Earsketch
Computer Vision: Neural Transfer Style & Green Screen Effect
Create Your First Chatbot with Rasa and Python
Building Custom Regional Reports with Google Analytics
Create a Business Marketing Brand Kit Using Canva
Introducción a los algoritmos de regresión
Support Vector Machine Classification in Python
Sentiment Analysis with Deep Learning using BERT
Support Vector Machine Classification in Python
University Admission Prediction Using Multiple Linear Regression
Advanced Features with Relational Database Tables Using SQLiteStudio
Predict Ideal Diamonds over Good Diamonds using a Random Forest using R
Create Informative Presentations with Google Slides
Intro to Time Series Analysis in R
Create Infographics using Piktochart
Statistical Data Visualization with Seaborn
Predictive Modelling with Azure Machine Learning Studio
Image Classification with CNNs using Keras
Classification with Transfer Learning in Keras
Exploratory Data Analysis with Seaborn
Regression with Automatic Differentiation in TensorFlow
Analyze Box Office Data with Seaborn and Python
Analyze Box Office Data with Plotly and Python
Logistic Regression with Python and Numpy
Deep Learning Inference with Azure ML Studio
Perform Sentiment Analysis with scikit-learn
Machine Learning Pipelines with Azure ML Studio
Build Random Forests in R with Azure ML Studio
Neural Network from Scratch in TensorFlow
Create Custom Layers in Keras
Deploy Models with TensorFlow Serving and Flask
Evaluate Machine Learning Models with Yellowbrick
Predict Employee Turnover with scikit-learn
Visual Machine Learning with Yellowbrick
Predicting House Prices with Regression using TensorFlow
Predict Sales Revenue with scikit-learn
Build Data Analysis tools using R and DPLYR
Image Compression with K-Means Clustering
Basic Sentiment Analysis with TensorFlow
Visualizing Citibike Trips with Tableau
Neural Style Transfer with TensorFlow
Simple Recurrent Neural Network with Keras
Image Noise Reduction with Auto-encoders using TensorFlow
Avoid Overfitting Using Regularization in TensorFlow
Save, Load and Export Models with Keras
Perform Feature Analysis with Yellowbrick
Regression Analysis with Yellowbrick
Data Visualization with Plotly Express
Analyze Text Data with Yellowbrick
Creating Custom Callbacks in Keras
Linear Regression with NumPy and Python
Multiple Linear Regression with scikit-learn
Image Data Augmentation with Keras
Video Basics with OpenCV and Python
Semantic Segmentation with Amazon Sagemaker
Object Detection with Amazon Sagemaker
Build an E-commerce Dashboard with Figma
Introduction to Relational Database and SQL
Create Customer Support Data with Google Sheets
Using TensorFlow with Amazon Sagemaker
Linear Regression with Python
Business Operations Support in Google Sheets
Named Entity Recognition using LSTMs with Keras
Build Multilayer Perceptron Models with Keras
Image Denoising Using AutoEncoders in Keras and Python
Create Online Employee Onboarding Course with Eduflow
Clustering Geolocation Data Intelligently in Python
Image Super Resolution Using Autoencoders in Keras
Traffic Sign Classification Using Deep Learning in Python/Keras
Computer Vision – Object Detection with OpenCV and Python
Computer Vision – Image Basics with OpenCV and Python
Support Vector Machines with scikit-learn
Siamese Network with Triplet Loss in Keras
Support Vector Machines in Python, From Start to Finish
Classification Trees in Python, From Start To Finish
Facial Expression Recognition with Keras
Spreadsheets for Beginners using Google Sheets
Predicting Salaries with Simple Linear Regression in R
Perform Real-Time Object Detection with YOLOv3
Understanding Deepfakes with Keras
Principal Component Analysis with NumPy
Graphing with ggplot2
Logistic Regression with NumPy and Python
Classify Radio Signals from Space using Keras
Building Candlestick Charts with Google Sheets
Image Classification with Amazon Sagemaker
Language Classification with Naive Bayes in Python
Create Technical Stock Charts Using R and Quantmod
